source/security/threatmonitoring/non-ukfast-install.md

```diff
@@ -0,0 +1,128 @@
+
+The provided commands will attempt to escalate to the privileged root user. 
+
+`sudo -i`
+
+If your current user is not able to escalate it's permissions to the root user or is not able to run the `sudo` command, please re-try from a different account with the correct permissions. Alternatively, you may run the commands using the root user directly.
+
+When you have root access, please continue to install our UKFast Defensive Security yum or apt package repository onto your server and then install our Threat Monitoring package and its dependencies via the commands shown on your MyUKFast page. An example for a CentOS 7 installation is shown below.
+
+```
+[root@my-server ~]$ rpm --import https://repo.thmon.ukfast.co.uk/key/UKFDEFSEC-GPG-KEY && \
+cat > /etc/yum.repos.d/ukfast-defensive-security.repo <<\EOF
+[ukfast_defensive_security]
+gpgcheck=1
+pgkey=https://repo.thmon.ukfast.co.uk/key/UKFDEFSEC-GPG-KEY
+enabled=1
+name=UKFast Defensive Security repository
+baseurl=https://repo.thmon.ukfast.co.uk/yum/rhel-centos-6
+EOF
+```
+
+```
+[root@my-server ~]$ yum install threat-monitoring -y
+```
+
+Once the package has been installed successfully, you need to perform 1 final step to link your server up to our Threat Monitoring infrastructure. 
+
+Before we do that, you have the option of setting a custom friendly that will later be shown in any Threat Monitoring dashboards when referring to this server. You can either specify this optional manually using the `-name=NAME_HERE` flag, or by entering your desired server in the `Name` text box provided on the MyUKfast installation page. Updating this text box will automatically update the commands that we need to run next.
+
+Finally, run the provided command to complete your install. Upon completion, you should see a success message reading, `Success! Your Threat Monitoring Agent is now configured and working`. 
+
+An example MyUKFast configuration setup is shown below.
+
+![configuration-and-install](files/setup-instructions-and-name.png)
+
+If you do encounter any issues during installation, please get in contact with UKFast support.
+
+## 4) Setup email contacts
+
+To receive email alerts for suspicious events, you will need to configure email alerts via MyUKFast. To do this, go to the "Alerts Manager" section under "My Account", as shown below.
+
+![setup-type](files/setup-alerts-manager.png)
+
+On the Alerts Manager page, select the "Server" tab. On here you will see a table of contacts and the types of alerts they have been configured to receive. 
+
+To set a contact to receive Threat Monitoring alerts, press the letter icon next to their name under the Threat Monitoring column and then press the "Save Changes" button, an example is shown below.
+
+![setup-type](files/setup-contacts.png)
+
+
+```eval_rst
+   .. title:: Installing Threat Monitoring on a non-UKFast server
+   .. meta::
```

Comment:
This title is not within the correct lengths needs to be maximum of 42 characters

not sure why travis not telling you will need to check that